### PR TITLE
rococo-runtime: Switch to latest `construct_runtime!` syntax

### PR DESCRIPTION
**Pull request by bkchr**
Besides that it fixes pallet macro errors in other crates that popped up
because of this switch.

### CODE REVIEW

File: runtime/rococo/src/validator_manager.rs [hunk 1]
```diff
@@ -16,84 +16,87 @@
...
 
-decl_error! {
-	pub enum Error for Module<T: Config> {}
-}
+		/// Privileged origin that can add or remove validators.
+		type PrivilegedOrigin: EnsureOrigin<<Self as frame_system::Config>::Origin>;
+	}
 
-decl_storage! {
-	trait Store for Module<T: Config> as ParachainProposer {
```
Comment (thiolliere) [inline runtime/rococo/src/validator_manager.rs:48]: the pallet prefix is changed from ParachainProposer to ValidatorManager, I guess we should migrate the storage

Comment (bkchr) [reply runtime/rococo/src/validator_manager.rs:48]: Yeah I know, but the storage items are just for storing temporary data. Aka, we don't really need to migrate them.

Comment (thiolliere) [reply runtime/rococo/src/validator_manager.rs:48]: ok then all good to me

**Review by thiolliere — APPROVED** [review]

**Comment by thiolliere** [comment]
I checked the metadata and there is no difference apart from:
* prefix changing for validator manager pallet, but this is known to be fine
* some call with no variant in beefy pallet, which is fine
* more information on storage on some pallets

**Pull request merged**